### PR TITLE
adds CLO apps and GitVote

### PR DESCRIPTION
**Pull request by RobertKielty**
Adds _my_ descriptions of the CLO suite of apps and Git Vote.

Deliberately looking for considered, reviews for tone, style, and messaging.

### CODE REVIEW

**Review by amye — APPROVED** [review]

**Review by tegioz — APPROVED** [review]

**Review by cynthia-sg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -115,6 +115,10 @@ CNCF Projects may freely select their own tools, produce their own documentation
 - [Scarf](https://scarf.sh) for advanced analytics for container & artifact distribution, package installation, and web traffic to source documentation
 - [Peritus.ai](https://peritus.ai/) for Machine Learning analytics and self-service for developer communities
 - [Curiefense](https://curiefense.io) for application layer protection (web/API) - WAF, DDoS, Rate limiting, and more
+- [CLOMonitor](https://clomonitor.io) scans repositories daily, checking for adherence to a range of best practises providing a way for end users to assess the health of open source projects from multiple points of view
```
Comment (krook) [inline README.md:118]: Is our standard `en-us` or `en-ie` for the word "practices"? 😊

**Review by onlydole — APPROVED** [review]
LGTM 👍🏼

File: README.md (hunk 1 above)
Comment (nate-double-u) [reply README.md:118]: I think we usually default to `en-us`

Comment (nate-double-u) [reply README.md:118]: (that said, [our style guide](https://github.com/cncf/foundation/blob/main/style-guide.md) is pretty sparse)

Comment (krook) [reply README.md:118]: I love that that style guide definitively addresses the [long-standing open source and cloud native hyphen tiff we've had with Grammarly](https://twitter.com/DanielKrook/status/1347606838632734723)! 

On the practices thing... we have already [set the precedent here in the README](https://github.com/cncf/servicedesk#program-management) so let's go with west-pond style.

Comment (RobertKielty) [reply README.md:118]: Hiberno English is the best English, the plurality of practises defendable here, I think, by the wide range of actions required (and checked) to make a project truly great 😜 

Happy to en-us it mind. Nice catch!

**Pull request merged**